October upload bypass exec (#12272)
rapid7/metasploit-framework: documentation/modules/exploit/multi/http/october_upload_bypass_exec.md

## Description
An authenticated user with permission to upload and manage media contents can
upload various files on the server. The application prevents the user from

> uploading PHP code by checking the file extension. It uses black-list based

Reviewer: ```suggestion
uploading PHP code by checking the file extension. It uses blacklist based
```

approach, as seen in octobercms/vendor/october/rain/src/Filesystem/
Definitions.php:blockedExtensions().

> ## Vulnerable  Sorftware

Reviewer: ```suggestion
## Vulnerable Software
```

> https://www.exploit-db.com/apps/4ff8a9688f31b7338020d0bc85da13fc-october-1.0.412.tar.gz

Reviewer: The `Vulnerable Software` section should make reference to the vulnerable version (`v1.0.412 (build 412)`), and whether previous versions were affected.

Reviewer 2: Handled this before landing.

## Author
Anti Räis (Discovery)
Touhid M.Shaikh (Metasploit Module)

## References
https://www.exploit-db.com/exploits/41936

## Tested on

> HackTheBox October Machine (IP: 10.10.10.16)

Reviewer: ```suggestion
This module was tested on October CMS version v1.0.412 on Ubuntu.
```
